test/cpp/qps/server.h

```diff
@@ -75,9 +77,20 @@ class Server {
     stats.set_idle_cpu_time(timer_result.idle_cpu_time);
     stats.set_cq_poll_count(poll_count);
     CoreStatsToProto(core_stats, stats.mutable_core_stats());
+
+    if (gpr_atm_full_cas(&marked_, static_cast<gpr_atm>(0),
```

Comment:
Why do we need both the atm and the event? Can we not just gpr_event_set mark_received_ here?

Reply:
You're right. I should just use gpr_event_get in the async version of the code. I will fix this.